### PR TITLE
Gem loading issue.

### PR DESCRIPTION
**Pull request by heaven**
Hi, I am having the problem with running this gem because I have a model called Client. So path in your gem should be specified explicitly.

Thanks,
Alex

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ doc
 spec/fixtures/calais.yml
 .bundle
 Gemfile.lock
+/.idea/
```
Comment (abhay) [inline .gitignore]: You should have this in your global .gitignore.

File: lib/calais.rb [hunk 2]
```diff
@@ -17,9 +17,9 @@
 
 $:.unshift File.expand_path(File.dirname(__FILE__)) + '/calais'
 
-require 'client'
-require 'response'
-require 'error'
+require 'calais/client'
+require 'calais/response'
+require 'calais/error'
```
Comment (abhay) [inline lib/calais.rb:20]: Remove the unshift above with your patch and I'm happy to merge this.

**Comment by heaven** [comment]
Done.

**Comment by abhay** [comment]
Unfortunately, this doesn't work as the loadpath doesn't find the files any more. I'll fix it. Thanks for the suggestion.

**Pull request closed without merging**